### ISSUE TITLE
time to release a new version!

### ISSUE THREAD

**Issue by faxmodem** [meltwater/puppet-cpan#27]
what do you think?

**Comment by faxm0dem** [comment]
sorry to insist, but it's been a while.
I'd be happy to help if you lack time in maintaining this module

**Comment by westbywest** [comment]
Bumping this issue

**Comment by steveames** [comment]
Really need an update sent to the forge. The version there doesn't work with Puppet 4 at all. I've taken to just pointing my Puppetfile at this repo but that's dicey since it could change...

**Comment by salimane** [comment]
@steveames @faxmodem @westbywest a pull request would be welcomed :)

**Comment by faxm0dem** [comment]
@salimane we can't do a release without write access to the repo

**Comment by salimane** [comment]
@faxm0dem I'm not saying a release :) You could fork the repo, make some changes and send a pull request

**Comment by faxm0dem** [comment]
@salimane sure thing, but what? do you need something in particular?

**Comment by salimane** [comment]
I was referring to @steveames comment :)
```
Really need an update sent to the forge. The version there doesn't work with Puppet 4 at all. I've taken to just pointing my Puppetfile at this repo but that's dicey since it could change...
```

**Comment by faxm0dem** [comment]
from what I understood in the comment you're referring to, the master branch actually does work with puppet4.

The reason I created this issue in the first place was simply to kindly ask @meltwater to add a new release tag and push it to puppet forge.

Then again indeed we could fork and release our own to puppet labs, but I'll wait a little bit more for the sake of courtesy.

**Comment by salimane** [comment]
@faxmodem ok :) I will release a new version tomorrow, don't worry :)

**Comment by faxm0dem** [comment]
\o/ that's awesome thanks

**Comment by westbywest** [comment]
Confirming @steveames comment that the current commit to this repo works fine under Puppet v4.5, the (soon to be old) version published to forge does not.  Also looking forward to being able to fix my Puppetfile to no longer point directly to this repo.  However, do please note current version published to forge is v1.0.1, whilst the metadata.json at current commit here is v1.0.0. I've added PR  https://github.com/meltwater/puppet-cpan/pull/30 incrementing the version to v1.0.2, if you need.  Thank you.

**Comment by salimane** [comment]
a new version v2.0.0 has been published at https://forge.puppet.com/meltwater/cpan .
Thanks everyone
